minio/mcs
# Add Watch api and UI integration

Uses a similar approach to Trace and Console Logs by using
websockets. It also includes the integration with the UI which
needs 3 input fields that are sent as query parameters.

<img width="1555" alt="Screen Shot 2020-05-14 at 10 13 22 AM" src="https://user-images.githubusercontent.com/11819101/81964736-e02a7500-95cb-11ea-873b-f9ea62dc67ae.png">

It includes: temporary dependencies
> Wait until a new release of mc comes which needs https://github.com/minio/mc/commit/a6d82862cd0206dd3574a24b072fc2995c027d28

To Test:
- Go into the new window for `Watch`.
- A bucket list should be rendered on the `Select` button.
- Prefix and suffix are optional, if not defined, watch will be done on all prefixes and/or suffixes.
- Once Start is clicked, it should start listening.
- Copy a file into the bucket that you selected (using mc) and only the events that match your choices
should appear on mcs.
- If you go to other window, watch should stop
